### PR TITLE
feat(crux-ui): playwright project

### PR DESCRIPTION
**Pull request by robot9706**
- Rearranged playwright tests into projects which depend on eachother, so unecessary tests won't run.
- Adjusted `maxFailures` so 1 failure terminates testing.
- Updated Playwright from `1.31.1` to `1.35.0`

### CODE REVIEW

File: web/crux-ui/e2e/utils/global.teardown.spec.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { test } from '@playwright/test'
+import { globalTeardown } from './global.teardown'
+
+test('global teardown', async () => {
```
Comment (polaroi8d) [inline web/crux-ui/e2e/utils/global.teardown.spec.ts:4]: What is the test here?

File: web/crux-ui/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "start": "npm run dev",
     "start:prod": "next start",
     "test:e2e": "playwright test",
-    "test:e2e-debug": "playwright test --debug",
+    "test:e2e-debug": "DEBUG=1 playwright test --debug",
```
Comment (polaroi8d) [inline web/crux-ui/package.json:27]: <3

**Review by polaroi8d — APPROVED** [review]
Small adjustments are okay for me, but please wait for @m8vago's review.

File: web/crux-ui/e2e/with-login/nodes-deploy.spec.ts [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+import { deployWithDagent } from '../utils/node-helper'
+import {
+  addDeploymentToVersionlessProject,
+  addImageToVersionlessProject,
+  createImage,
+  createProject,
+  createVersion,
+} from '../utils/projects'
+
+const prefix = 'first'
```
[the remark this answers is not in the document]
Comment (robot9706) [reply web/crux-ui/e2e/with-login/nodes-deploy.spec.ts:13]: Copied these from other test files, but yes we should use uppercase consts.

**Review by m8vago — CHANGES_REQUESTED** [review]

File: web/crux-ui/playwright.config.ts [hunk 4]
```diff
@@ -41,16 +60,41 @@ const config: PlaywrightTestConfig = {
...
+    },
+    {
+      name: 'test-without-login',
+      testMatch: /without\-login\/.*spec\.ts/,
+      dependencies: ['global-setup'],
       use: {
         ...devices['Desktop Chromium'],
       },
     },
+    createProject('test-registry', 'with-login/registry.spec.ts'),
```
[the remark this answers is not in the document]
Comment (robot9706) [reply web/crux-ui/playwright.config.ts]: Not really :)

**Pull request merged**